### DISCUSSION TITLE
DAO unsynchronized / seed nodes hampered

### DISCUSSION THREAD

**Discussion by lukode33** [bisq-network/bisq#7018] [General]
Good evening everyone! Lately on Telegram support channel and matrix chat a lot of people are complaining about DAO being unsynchronized with their clients, errors regarding the consensus and general halt of the trading platform that became partially unusable. I wanted to open this discussion in order to receive any updates, details and debugging (if any) regarding the issue that devs and seed nodes operators are encountering. Peace out ✌️

**Comment by ghost** [comment]
Two issues here: one is caused whenever the Bitcoin blockchain has a [temporary fork](https://forkmonitor.info/nodes/btc) which can disrupt seednodes for several hours.  The other, cause unknown, results in the Bisq application (DAO state) becoming out of consensus with its peers - this one seems to have worsened in the past weeks although the software has not changed.  Both have been known bugs for some time but no solution has been found.

**Comment by lukode33** [comment]
> The other, cause unknown, results in the Bisq application (DAO state) becoming out of consensus with its peers - this one seems to have worsened in the past weeks although the software has not changed.

Thank you for the reply @jmacxx. Considering that the software has not changed, do you think that the issue is fixable by reorganizing how seed nodes reach the consensus and share it with other network peers? If not what other viable ways are devs following to fix this problem? Thank you in advance :)

**Comment by ghost** [comment]
My opinion is that any likely fix would be implemented in [Bisq2](https://github.com/bisq-network/bisq2/discussions).  That allows a clean slate to address the unsolvable issues encountered in Bisq.
